package DBoW2.proto;
import "brief_node.proto";
import "brief_word.proto";

message BriefVocabulary {
  optional int32 k = 1;
  optional int32 l = 2;
  optional int32 scoring_type = 3;
  optional int32 weighting_type = 4;
  repeated BriefNode nodes = 5;
  repeated BriefWord words = 6;
}